<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomLampsDialogFrame</class>
 <widget class="QFrame" name="CustomLampsDialogFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>494</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QGroupBox" name="d_groupBoxPosition">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>110</height>
        </size>
       </property>
       <property name="title">
        <string>Position</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="d_labelColumnPosition">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Pos column:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="d_spinBoxColumnPosition"/>
          </item>
          <item>
           <widget class="QLabel" name="d_labelRowPosition">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Pos row:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="d_spinBoxRowPosition"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="d_labelColumnDimension">
            <property name="minimumSize">
             <size>
              <width>70</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Dim column:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="d_spinBoxColumnDimension">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="d_labelRowDimension">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>70</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Dim row:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="d_spinBoxRowDimension">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="d_stackedWidgetStancil">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="d_firstPage">
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QGroupBox" name="d_groupBoxStancil">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>Stancil</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QListView" name="d_listViewStancil">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="d_pushButtonAddStancil">
                  <property name="maximumSize">
                   <size>
                    <width>45</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Add</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/Icons/addTab.png</normaloff>:/Icons/addTab.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="d_pushButtonDeleteStancil">
                  <property name="maximumSize">
                   <size>
                    <width>45</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Delete</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>:/Icons/removeTab.png</normaloff>:/Icons/removeTab.png</iconset>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="d_pushButtonChangeStancil">
                  <property name="maximumSize">
                   <size>
                    <width>45</width>
                    <height>23</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Change</string>
                  </property>
                  <property name="icon">
                   <iconset resource="../Icons.qrc">
                    <normaloff>:/Icons/refresh.png</normaloff>:/Icons/refresh.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="d_labelPreviewStancil">
                <property name="text">
                 <string>Preview:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CustomLampsGraphicsView" name="d_widgetViewStancil" native="true">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="d_secondPage">
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QGroupBox" name="d_groupBoxStancilInput">
           <property name="title">
            <string>New Stancil</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_10">
              <item>
               <widget class="QLineEdit" name="d_lineEditStancilName">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="placeholderText">
                 <string>Stancil Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPlainTextEdit" name="d_plainTextEditStancil">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>(+0.1/0.1;0.9/0.5;0.1/0.9;)
(-0.4/0.4;0.6/0.5;0.4/0.6;)</string>
                </property>
                <property name="placeholderText">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QPushButton" name="d_pushButtonStancilOk">
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>OK</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="d_pushButtonStancilCancel">
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Cancel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="d_labelPreviewStancilInput">
                <property name="text">
                 <string>Preview:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="CustomLampsGraphicsView" name="d_widgetViewStancilInput" native="true">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>100</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomLampsGraphicsView</class>
   <extends>QWidget</extends>
   <header>LampsGraphicsView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections/>
</ui>
